docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -506,6 +506,93 @@ Different KDFs are suitable for different tasks such as:
         ``key_material`` generates the same key as the ``expected_key``, and
         raises an exception if they do not match.
 
+.. currentmodule:: cryptography.hazmat.primitives.kdf.x963kdf
+
+.. class:: X963KDF(algorithm, length, otherinfo, backend)
+
+    .. versionadded:: 1.0
+
+    X963KDF (ANSI X9.63 Key Derivation Function) is defined by ANSI
+    X9.63-2001 `ANSI X9.63:2001`_ document, to be used to
+    derive keys for use after a Key Exchange negotiation operation.
```

Comment:
Sure, how about "SEC 1 v2.0(link) recommends that ConcatKDF(link to our docs for ConcatKDF) be used for new projects. This KDF should only be used for backwards compatibility with pre-existing implementations."